package.json

```diff
@@ -39,55 +41,87 @@
     "url": "https://github.com/nolimits4web/skeleton-elements/issues"
   },
   "homepage": "https://github.com/nolimits4web/skeleton-elements#readme",
+  "dependencies": {
+    "@angular/animations": "~10.1.0",
+    "@angular/common": "~10.1.0",
+    "@angular/compiler": "~10.1.0",
+    "@angular/core": "~10.1.0",
+    "@angular/forms": "~10.1.0",
+    "@angular/platform-browser": "~10.1.0",
+    "@angular/platform-browser-dynamic": "~10.1.0",
+    "@angular/router": "~10.1.0",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.2"
+  },
```

Comment:
Do we need all of this in dependencies and not in devDependencies?

Reply:
No, I just thought it's an internal package.json and it's not really matters. It can be moved to dev deps.